### PR TITLE
Clarify Toast duration Documentation.

### PR DESCRIPTION
**Pull request by ignacio-chiazzo**
The documentation doesn't clarify how duration works. 

Link: https://capacitor.ionicframework.com/docs/apis/toast/

### CODE REVIEW

**Comment by ignacio-chiazzo** [comment]
Thanks for the quick review @danielsogl.    The document you referenced has a link to 
`<plugin-api name="toast"></plugin-api>` so I edited the schema.  I based on this code: https://github.com/ionic-team/capacitor/blob/4a1c19f361641df8ebee4b624d1f9c1f6de50548/core/src/web/toast.ts#L13-L22.

File: core/src/core-plugin-definitions.ts [hunk 1]
```diff
@@ -1612,6 +1612,9 @@ export interface ToastPlugin extends Plugin {
 
 export interface ToastShowOptions {
   text: string;
+    /**
```
Comment (jcesarmobile) [inline core/src/core-plugin-definitions.ts]: Can you align the * in the comments?

```suggestion
  /**
```

**Comment by jcesarmobile** [comment]
Just checked the source code and looks like every platform if using different values. 

For android it's 2000-3500 (and can't be changed because those are Android constants)
For iOS it's 1500-3000
For web/electron it's 3000-5000.

So I think first the iOS and web values should be changed to match Android ones (as those can't be changed) and then document those values.

Sorry for requesting changes before checking.

**Comment by jcesarmobile** [comment]
closing until they all have the same value https://github.com/ionic-team/capacitor/issues/2175

**Pull request closed without merging**